webob/request.py

```diff
@@ -161,9 +176,12 @@ def encget(self, key, default=NoDefault, encattr=None):
             if not encattr:
                 return val
             encoding = getattr(self, encattr)
-            if encoding in _LATIN_ENCODINGS: # shortcut
+            if encoding in _LATIN_ENCODINGS:  # shortcut
                 return val
-            return bytes_(val, 'latin-1').decode(encoding)
+            try:
+                return bytes_(val, 'latin-1').decode(encoding)
```

Comment:
The default for encoding here is `UTF-8` as defined by `Request.url_encoding` (`encget` is used only by `environ_decoder`)

https://github.com/Pylons/webob/blob/master/webob/request.py#L313

What I would like to see is if an attempt to decode using `UTF-8` fails, that we fall back to trying to decode using `latin-1`.

WebOb expects that requests are only be submitted as `UTF-8`, the HTTP standard states that the only valid encoding is `ISO-8859-1`, so I don't think it's correct to treat everything as if it is `UTF-8`.